spec/cli/navigate_spec.rb

```diff
@@ -128,6 +128,14 @@ def require_relative(path)
     it "should allow '=' in string" do
       Razor::CLI::Parse.new(['create-repo', '--name=\'with=equals\'', '--url', 'http://url.com/some.iso', '--task', 'noop']).navigate.get_document['name'].should =~ /with=equals/
     end
+
+    it "should not allow double-dash with single character flag" do
+      nav = Razor::CLI::Parse.new(['create-broker', '--name=some-broker', '--broker-type', 'puppet-pe', '--c', 'server=abc.com']).navigate
+      expect{nav.get_document}.to raise_error(ArgumentError, 'Unexpected argument --c')
+    end
+    it "should allow single-dash with single character flag" do
+      Razor::CLI::Parse.new(['create-broker', '--name=some-broker', '--broker-type', 'puppet-pe', '-c', 'server=abc.com']).navigate.get_document['configuration']['server'].should == 'abc.com'
+    end
```

Comment:
What about testing that the right error gets thrown for a long flag with a single dash?